wp-content/themes/oceanwp-child/functions.php

```diff
@@ -413,21 +413,21 @@ function add_action_initiatives_by_region() {
 							}else{
 
 								foreach($action as $key => $take_action){
-
+									
 									?>
-									<p class="action-type">
-									<i class="fa-solid fa-circle-exclamation"></i>	<?php echo esc_html($take_action->name); ?></p>
-								<?php
-
+									<p class="action-type"> 
+									<img src="../../wp-content/uploads/icons/exclamationmark.svg" class="icons"/>	<?php echo esc_html($take_action->name); ?></p>
```

Comment:
I don't think we should do this like that.
Keep using the `i` as it is semantically correct.
Create a class per icon type like so:
```
.ggrc-icon-exclamation-mark {
  &::after {
    content: url('/wp-content/uploads/icons/exclamationmark.svg'); // not sure the path is correct, need to verify
    display: block;
    height: 1.5em;
    width: 1.5em;
  }
}
```
Then it is reusable everywhere.